.github/workflows/base_schema.yml

```diff
@@ -61,8 +61,10 @@ jobs:
       run: |
         for k in $(yq '. | keys | .[]' descriptions.yml); do
           newDescription=$(yq ".\"$k\"" descriptions.yml);
+          newSummary=$(yq ".\"$k\"" summary.yml);
```

Comment:
You probably want to do this in separate `for` loop.

Reply:
@bojanrajh , Yes I tried that too but the workflow failed to process : https://github.com/shopware/admin-api-reference/actions/runs/8465370243/job/23191984692

Reply:
Can you please try removing second summary `".paths./_action/number-range/reserve/{type}/{saleschannel}.get"` as it contains `{` and it might break the script: https://github.com/shopware/admin-api-reference/pull/167/files#diff-a5e5f2031c70ff6c3b7af0db03df31c28c31b6988b8a5b731c76b82770e3baa2R4 ?
Or maybe try escaping `{` and `}` with `\{` and `\}` respectively?